android-components/components/feature/prompts/src/main/java/mozilla/components/feature/prompts/identitycredential/PrivacyPolicyDialogFragment.kt

```diff
@@ -37,7 +37,7 @@ internal class PrivacyPolicyDialogFragment : AbstractPromptTextDialogFragment()
         val builder = AlertDialog.Builder(requireContext())
             .setTitle(title)
             .setCancelable(true)
-            .setPositiveButton(android.R.string.ok) { _, _ ->
+            .setPositiveButton(R.string.mozac_feature_prompts_identity_credentials_continue) { _, _ ->
                 onConfirmAction(true)
             }
             .setNegativeButton(android.R.string.cancel) { _, _ ->
```

Comment:
I think we should also make sure this string is localized instead of deferring to the android strings.